app.js

```diff
@@ -1,26 +1,24 @@
-var express = require('express'),
-  reqlib = require('app-root-path').require,
-  logger = require('morgan'),
-  cookieParser = require('cookie-parser'),
-  bodyParser = require('body-parser'),
-  app = express(),
-  fs = require('fs'),
-  SerialPort = require('serialport'),
-  jsonStore = reqlib('/lib/jsonStore.js'),
-  cors = require('cors'),
-  ZWaveClient = reqlib('/lib/ZwaveClient'),
-  MqttClient = reqlib('/lib/MqttClient'),
-  Gateway = reqlib('/lib/Gateway'),
-  store = reqlib('config/store.js'),
-  config = reqlib('config/app.js'),
-  debug = reqlib('/lib/debug')('App'),
-  history = require('connect-history-api-fallback'),
-  utils = reqlib('/lib/utils.js');
+const express = require('express')
```

Comment:
switching to `const` is unfortunately going to cause pain when we write tests for this and we need to mock dependencies in to unit test. can we stick to `var` or `let` ?

Reply:
If it causes problems when writing tests I will use var instead, let doesn't have any sense as them are global